spec/services/rapid_ready_for_decision/constants_spec.rb

```diff
@@ -5,27 +5,33 @@
 RSpec.describe RapidReadyForDecision::Constants do
   let(:form526_submission) { create(:form526_submission, :hypertension_claim_for_increase) }
 
-  describe '.extract_disability_constant_list' do
-    subject { described_class.extract_disability_constant_list(form526_submission) }
+  describe 'processor_class' do
+    it 'resolves all processor_class values to classes' do
+      classes = RapidReadyForDecision::Constants::DISABILITIES.values.pluck(:processor_class).map(&:constantize)
+      expect(classes.any?(&:nil?)).to eq false
```

Comment:
This is so good.

Although I think a non-resolvable name will raise `NameError` on the previous line. Anyway, the test works.